virtcontainers/kata_agent.go

```diff
@@ -1569,6 +1569,13 @@ func (k *kataAgent) sendReq(request interface{}) (interface{}, error) {
 	span.SetTag("request", request)
 	defer span.Finish()
 
+	if k.state.ProxyPid > 0 {
```

Comment:
Same thing here, I don't think this is related to cgroups, and this should not be part the same PR.